src/traces/parcoords/lines.js

```diff
@@ -128,8 +126,8 @@ function makePoints(sampleCount, dims, color) {
         for(var k = 0; k < 64; k++) {
             points.push(
                 k < len ? dims[k].paddedUnitValues[j] :
-                    k === 64 - 1 ? adjustDepth(color[j]) :
-                        k >= 64 - 4 ? calcPickColor(j, 64 - 2 - k) : 0.5
+                    k === 63 ? adjustDepth(color[j]) :
+                        k > 59 ? calcPickColor(j, 62 - k) : 0.5
```

Comment:
Can we turn those `64`, `63`, `62` and `59` numbers into constants to make it easier to recognise? Thanks!

Reply:
GitHub duplicated this comment.
I resolve this instance.